src/operator/tensor/la_op.cc

```diff
@@ -983,6 +984,7 @@ Examples::
 .add_argument("A", "NDArray-or-Symbol", "Tensor of square matrix");
 
 NNVM_REGISTER_OP(_backward_linalg_det)
+.add_alias("_backward_np_linalg_det")
```

Comment:
Adding np alias for backward op seems not necessary.

Reply:
Fixed. Thx.